src/cls/IPM/Storage/SystemRequirements.cls

```diff
@@ -19,16 +19,20 @@ Method VersionStringSet(tValue) As %Status
 
 Property Interoperability As %String(VALUELIST = ",enabled,disabled", XMLPROJECTION = "ATTRIBUTE");
 
+Property Health As %String(VALUELIST = ",true,false", XMLPROJECTION = "ATTRIBUTE");
```

Comment:
Why use string when you can use a boolean?
Same for Interoperability above.

Reply:
Changing Interoperability would be backwards-incompatible